core/src/main/kotlin/io/rover/campaigns/core/tracking/AutoScreenTracker.kt

```diff
@@ -0,0 +1,50 @@
+package io.rover.campaigns.core.tracking
+
+import android.app.Activity
+import android.app.Application
+import android.content.pm.PackageManager
+import android.os.Bundle
+import io.rover.campaigns.core.RoverCampaigns
+import io.rover.campaigns.core.events.EventQueueServiceInterface
+import io.rover.campaigns.core.logging.log
+import java.lang.Exception
+
+private const val TM_PACKAGE_PREFIX = "com.ticketmaster"
+private const val ROVER_PACKAGE_PREFIX = "io.rover"
+
+const val TRACKING_LABEL_KEY = "rvAutoTrackingLabelKey"
+const val AUTO_TRACKING_ENABLED_KEY = "rvAutoTrackingEnabled"
+const val ACTIVITY_EXCLUDE_FROM_TRACKING = "rvAutoTrackingExcludeActivity"
+
+internal class AutoScreenTracker : Application.ActivityLifecycleCallbacks {
+    override fun onActivityPaused(activity: Activity?) {}
+
+    override fun onActivityResumed(activity: Activity?) {
+        activity?.let {
```

Comment:
I think this block is too long for `it`s.  Maybe assign it the name `activity`? Pretty sure you can shadow outer scope names within Kotlin blocks.